[W 2024-02-13 13:30:05.503 ServerApp] A `_jupyter_server_extension_points` function was not found in notebook_shim. Instead, a `_jupyter_server_extension_paths` function was found and will be used for now. This function name will be deprecated in future releases of Jupyter Server.
[I 2024-02-13 13:30:05.505 ServerApp] jupyter_lsp | extension was successfully linked.
[I 2024-02-13 13:30:05.517 ServerApp] jupyter_server_terminals | extension was successfully linked.
[I 2024-02-13 13:30:05.529 ServerApp] jupyterlab | extension was successfully linked.
[I 2024-02-13 13:30:05.540 ServerApp] notebook | extension was successfully linked.
[I 2024-02-13 13:30:05.792 ServerApp] notebook_shim | extension was successfully linked.
[I 2024-02-13 13:30:05.830 ServerApp] notebook_shim | extension was successfully loaded.
[I 2024-02-13 13:30:05.832 ServerApp] jupyter_lsp | extension was successfully loaded.
[I 2024-02-13 13:30:05.833 ServerApp] jupyter_server_terminals | extension was successfully loaded.
[I 2024-02-13 13:30:05.836 LabApp] JupyterLab extension loaded from /export/scratch2/home/savvina/new_environment/conda/envs/elliot_env/lib/python3.8/site-packages/jupyterlab
[I 2024-02-13 13:30:05.836 LabApp] JupyterLab application directory is /export/scratch2/home/savvina/new_environment/conda/envs/elliot_env/share/jupyter/lab
[I 2024-02-13 13:30:05.837 LabApp] Extension Manager is 'pypi'.
[I 2024-02-13 13:30:05.840 ServerApp] jupyterlab | extension was successfully loaded.
[I 2024-02-13 13:30:05.843 ServerApp] notebook | extension was successfully loaded.
[I 2024-02-13 13:30:05.844 ServerApp] Serving notebooks from local directory: /export/scratch2/home/savvina/new_environment/Elliot
[I 2024-02-13 13:30:05.844 ServerApp] Jupyter Server 2.12.5 is running at:
[I 2024-02-13 13:30:05.844 ServerApp] http://localhost:8888/tree?token=380f82803d9189adec861a1cc64c7f3a0e302727398d7326
[I 2024-02-13 13:30:05.844 ServerApp]     http://127.0.0.1:8888/tree?token=380f82803d9189adec861a1cc64c7f3a0e302727398d7326
[I 2024-02-13 13:30:05.844 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 2024-02-13 13:30:05.847 ServerApp] 
    
    To access the server, open this file in a browser:
        file:///export/scratch2/home/savvina/.local/share/jupyter/runtime/jpserver-60446-open.html
    Or copy and paste one of these URLs:
        http://localhost:8888/tree?token=380f82803d9189adec861a1cc64c7f3a0e302727398d7326
        http://127.0.0.1:8888/tree?token=380f82803d9189adec861a1cc64c7f3a0e302727398d7326
[I 2024-02-13 13:30:07.897 ServerApp] Skipped non-installed server(s): bash-language-server, dockerfile-language-server-nodejs, javascript-typescript-langserver, jedi-language-server, julia-language-server, pyright, python-language-server, python-lsp-server, r-languageserver, sql-language-server, texlab, typescript-language-server, unified-language-server, vscode-css-languageserver-bin, vscode-html-languageserver-bin, vscode-json-languageserver-bin, yaml-language-server
[I 2024-02-13 13:30:47.131 ServerApp] 302 GET /?token=[secret] (@127.0.0.1) 1.53ms
[W 2024-02-13 13:31:16.060 ServerApp] Clearing invalid/expired login cookie username-127-0-0-1-8888
[I 2024-02-13 13:31:16.061 JupyterNotebookApp] 302 GET /tree (@127.0.0.1) 1.90ms
[I 2024-02-13 13:31:25.850 ServerApp] User b05ce82a73d844b4a0a11c10edf6043f logged in.
[I 2024-02-13 13:31:25.851 ServerApp] 302 POST /login?next=%2Ftree (b05ce82a73d844b4a0a11c10edf6043f@127.0.0.1) 2.35ms
[I 2024-02-13 13:37:04.493 ServerApp] Creating new notebook in 
[I 2024-02-13 13:37:11.181 ServerApp] Kernel started: 710d78a4-6ba7-40ab-ab52-dc24877f2249
[I 2024-02-13 13:37:11.807 ServerApp] Connecting to kernel 710d78a4-6ba7-40ab-ab52-dc24877f2249.
[I 2024-02-13 13:39:06.849 ServerApp] Saving file at /First try.ipynb
[I 2024-02-13 13:41:06.952 ServerApp] Saving file at /First try.ipynb
[I 2024-02-13 13:43:07.015 ServerApp] Saving file at /First try.ipynb
[I 2024-02-13 13:45:07.062 ServerApp] Saving file at /First try.ipynb
[I 2024-02-13 13:46:42.759 ServerApp] Kernel restarted: 710d78a4-6ba7-40ab-ab52-dc24877f2249
[I 2024-02-13 13:46:42.775 ServerApp] Starting buffering for 710d78a4-6ba7-40ab-ab52-dc24877f2249:2ffafce9-586a-419c-8fa2-61c0d7d36157
[I 2024-02-13 13:46:42.792 ServerApp] Connecting to kernel 710d78a4-6ba7-40ab-ab52-dc24877f2249.
[I 2024-02-13 13:46:42.792 ServerApp] Restoring connection for 710d78a4-6ba7-40ab-ab52-dc24877f2249:2ffafce9-586a-419c-8fa2-61c0d7d36157
[I 2024-02-13 13:47:07.119 ServerApp] Saving file at /First try.ipynb
2024-02-13 13:48:27.340718: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2024-02-13 13:48:27.340764: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
[I 2024-02-13 13:49:07.172 ServerApp] Saving file at /First try.ipynb
[I 2024-02-13 13:51:08.065 ServerApp] Saving file at /First try.ipynb
[I 2024-02-13 13:53:09.057 ServerApp] Saving file at /First try.ipynb
[I 2024-02-13 13:53:34.215 ServerApp] Creating new file in /elliot/config_files
[I 2024-02-13 13:56:49.277 ServerApp] Copying First try.ipynb to 
[I 2024-02-13 13:56:52.210 ServerApp] Starting buffering for 710d78a4-6ba7-40ab-ab52-dc24877f2249:2ffafce9-586a-419c-8fa2-61c0d7d36157
[I 2024-02-13 13:56:52.830 ServerApp] Kernel started: 426060dc-bf09-4904-ab96-c21f6cda5563
[I 2024-02-13 13:56:53.513 ServerApp] Connecting to kernel 426060dc-bf09-4904-ab96-c21f6cda5563.
[I 2024-02-13 13:58:16.163 ServerApp] Saving file at /elliot/config_files/sample_hello_world.yml
[I 2024-02-13 13:58:18.149 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 13:58:52.724 ServerApp] Saving file at /Second try.ipynb
2024-02-13 13:59:14.998721: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2024-02-13 13:59:14.998770: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
[I 2024-02-13 13:59:37.506 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 14:00:52.780 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 14:02:52.843 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 14:04:51.141 ServerApp] Creating new directory in /elliot/data
[I 2024-02-13 14:05:17.335 ServerApp] Uploading file to /elliot/data/fairbook/fairbook_events.csv
[I 2024-02-13 14:05:17.396 ServerApp] Saving file at /elliot/data/fairbook/fairbook_events.csv
[I 2024-02-13 14:05:34.689 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 14:06:54.030 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 14:10:55.039 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 14:12:22.798 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 14:12:24.551 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 14:12:56.018 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 14:14:57.052 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 14:16:57.116 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 14:18:57.164 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 14:20:58.007 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 14:22:59.014 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 14:50:29.304 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 15:02:42.552 ServerApp] Starting buffering for 426060dc-bf09-4904-ab96-c21f6cda5563:540bd110-045c-4f2c-b2e8-0eabd649efc2
[I 2024-02-13 15:02:42.562 ServerApp] Connecting to kernel 426060dc-bf09-4904-ab96-c21f6cda5563.
[I 2024-02-13 15:02:42.562 ServerApp] Restoring connection for 426060dc-bf09-4904-ab96-c21f6cda5563:540bd110-045c-4f2c-b2e8-0eabd649efc2
[W 2024-02-13 15:04:12.572 ServerApp] WebSocket ping timeout after 90009 ms.
[I 2024-02-13 15:04:17.577 ServerApp] Starting buffering for 426060dc-bf09-4904-ab96-c21f6cda5563:540bd110-045c-4f2c-b2e8-0eabd649efc2
[I 2024-02-13 15:42:25.483 ServerApp] Connecting to kernel 426060dc-bf09-4904-ab96-c21f6cda5563.
[I 2024-02-13 15:42:25.483 ServerApp] Restoring connection for 426060dc-bf09-4904-ab96-c21f6cda5563:540bd110-045c-4f2c-b2e8-0eabd649efc2
[I 2024-02-13 15:42:33.225 ServerApp] Creating new file in 
[I 2024-02-13 15:42:50.861 ServerApp] Saving file at /requirements.txt
[I 2024-02-13 15:43:00.652 ServerApp] Saving file at /requirements.txt
[I 2024-02-13 15:43:16.479 ServerApp] Saving file at /requirements.txt
[I 2024-02-13 16:21:48.372 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 16:23:48.416 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 16:25:48.450 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 16:27:41.764 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 16:28:24.064 ServerApp] Creating new directory in /elliot/results
[I 2024-02-13 16:28:34.507 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 16:28:47.998 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 16:29:26.376 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 16:31:27.373 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 16:36:50.377 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 16:38:50.459 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 16:40:27.383 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 16:41:31.355 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 16:43:19.798 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 16:43:31.401 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-13 16:45:17.478 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-13 16:45:20.342 ServerApp] Saving file at /Second try.ipynb
[W 2024-02-13 16:47:25.487 ServerApp] WebSocket ping timeout after 119773 ms.
[I 2024-02-13 16:47:30.497 ServerApp] Starting buffering for 426060dc-bf09-4904-ab96-c21f6cda5563:540bd110-045c-4f2c-b2e8-0eabd649efc2
[I 2024-02-14 10:44:00.359 ServerApp] 302 GET / (@127.0.0.1) 0.92ms
[I 2024-02-14 10:46:48.364 ServerApp] Connecting to kernel 426060dc-bf09-4904-ab96-c21f6cda5563.
[I 2024-02-14 10:48:48.310 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 10:50:48.895 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 11:21:22.862 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 11:24:08.029 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-14 11:24:43.826 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-14 11:24:48.249 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
2024-02-14 11:25:10.974943: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2024-02-14 11:25:10.974987: W tensorflow/stream_executor/cuda/cuda_driver.cc:312] failed call to cuInit: UNKNOWN ERROR (303)
2024-02-14 11:25:10.975019: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (alice.ia.cwi.nl): /proc/driver/nvidia/version does not exist
2024-02-14 11:25:10.975217: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-02-14 11:25:10.990493: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 1996120000 Hz
2024-02-14 11:25:10.997363: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x3e05980 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2024-02-14 11:25:10.997420: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
[I 2024-02-14 11:25:23.825 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 11:27:23.884 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 11:29:23.945 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 11:31:24.003 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 11:33:24.074 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 11:35:24.141 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 11:37:24.213 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 11:39:24.808 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 11:40:22.806 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-14 11:43:25.794 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 11:54:25.185 ServerApp] Starting buffering for 426060dc-bf09-4904-ab96-c21f6cda5563:dc1a5691-9f1f-4c33-a021-eadca197e2e6
[I 2024-02-14 11:54:25.256 ServerApp] Connecting to kernel 426060dc-bf09-4904-ab96-c21f6cda5563.
[I 2024-02-14 11:54:25.257 ServerApp] Restoring connection for 426060dc-bf09-4904-ab96-c21f6cda5563:dc1a5691-9f1f-4c33-a021-eadca197e2e6
[W 2024-02-14 11:55:55.260 ServerApp] WebSocket ping timeout after 90003 ms.
[I 2024-02-14 11:56:00.269 ServerApp] Starting buffering for 426060dc-bf09-4904-ab96-c21f6cda5563:dc1a5691-9f1f-4c33-a021-eadca197e2e6
[I 2024-02-14 12:04:18.777 ServerApp] Connecting to kernel 426060dc-bf09-4904-ab96-c21f6cda5563.
[I 2024-02-14 12:04:47.957 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-14 12:32:38.172 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-14 12:34:38.217 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-14 12:34:43.121 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-14 12:36:01.579 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-14 12:36:10.433 ServerApp] Saving file at /elliot/config_files/savvina_config.yml
[I 2024-02-14 12:36:38.162 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 12:40:38.217 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 12:42:39.166 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 12:46:41.157 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 12:50:43.144 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 12:54:45.152 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 12:58:47.127 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:00:48.120 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:04:50.134 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:08:52.109 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:12:54.123 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:14:55.110 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:18:57.097 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:22:59.084 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:27:01.097 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:31:03.089 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:33:04.078 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:37:06.073 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:41:08.083 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:45:10.072 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:49:12.064 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:51:13.065 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:55:15.138 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 13:59:16.080 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 14:03:16.178 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 14:06:54.594 ServerApp] Creating new notebook in 
[I 2024-02-14 14:06:57.773 ServerApp] Kernel started: c788e460-9f22-48ed-b62e-2a3520c3bbcf
[I 2024-02-14 14:06:58.378 ServerApp] Connecting to kernel c788e460-9f22-48ed-b62e-2a3520c3bbcf.
[I 2024-02-14 14:07:17.042 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 14:08:58.017 ServerApp] Saving file at /PopularityGood.ipynb
[I 2024-02-14 14:09:18.083 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 14:13:20.019 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 14:16:23.329 ServerApp] Copying elliot/config_files/savvina_config.yml to /elliot/config_files
[I 2024-02-14 14:17:22.048 ServerApp] Saving file at /Second try.ipynb
[I 2024-02-14 14:19:42.909 ServerApp] Creating new directory in /elliot/data
